### PR TITLE
high_level: ml: Renamed `accuracy` to `score`

### PR DESCRIPTION
**Pull request by programmer290399**
* Fixes: #1181 
* Ready for review.

### CODE REVIEW

**Comment by programmer290399** [comment]
* Lint commit is failing, and it is due to some bug that I have [added in the enhancement 
 issue](https://github.com/intel/dffml/issues/1136#issuecomment-922460785) related to it. **The commit msg format is correct for all the commits.**

File: CHANGELOG.md [hunk 1]
```diff
@@ -160,6 +160,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - IntegrationCLITestCase creates a new directory and chdir into it for each test
 - Automated testing of Automating Classification tutorial
 - `dffml version` command now prints git repo hash and if the repo is dirty
+- `dffml.high_level.ml.accuracy` to `dffml.high_level.ml.score`
```
Comment (pdxjohnny) [inline CHANGELOG.md:163]: Let's put this in `unreleased`, 
```suggestion
- Renamed `accuracy()` to `score()`
```

Comment (programmer290399) [reply CHANGELOG.md:163]: 😅 my bad, added it here by mistake, would move it to unreleased and would also reword it as per your suggestion above

**Comment by pdxjohnny** [comment]
Great work! Thank you!

**Pull request merged**